webserver/views/api/v1/core.py

```diff
@@ -45,14 +45,14 @@ def get_high_level(mbid):
     an offset as a query parameter in the form of ?n=x, where x is an integer
     starting from 0
```

Comment:
"Use the `/mbid/count/` (link) endpoint to get the total number of documents for this mbid."